src/StateBridge.sol

```diff
@@ -101,9 +101,13 @@ contract StateBridge is FxBaseRootTunnel, Ownable2Step {
     /// @notice Sends the latest WorldID Identity Manager root to all chains.
     /// @dev Calls this method on the L1 Proxy contract to relay roots and timestamps to WorldID supported chains.
     /// @param root The latest WorldID Identity Manager root.
-    function sendRootMultichain(uint256 root) external onlyWorldIDIdentityManager {
+    /// @param opGasLimit The gas limit for the Optimism transaction (how much gas to buy on Optimism with the message)
+    function sendRootMultichain(uint256 root, uint32 opGasLimit)
```

Comment:
Could we make `opGasLimit` a parameter of this contract that can be modified by the owner?

It would have 2 benefits:
1. No need to redeploy - no changes except for in the state bridge
2. We wouldn't leak an implementation detail (Optimism gas limit) in the `registerIdentities` method

Reply:
It's a different gas limit for each of the three calls and needs to be a different call to eth_estimateGas. If  we want to be fixed cost then we should just keep it constant like before, but we are going to be overpaying.